Week2/homework/Masoud/js-exercises/EX3-theRecipeCard.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+let myMealRecipe = {};
+myMealRecipe = {
+  title : 'pizza',
+  servings : 2,
+  ingredients : ['cheese 1', 'flour 0.5', 'tomato 3', 'mozerrella 0.2']
+}
+
+// code with just for
+let propertyCaption = ['Meal name', 'Serves', 'Ingredients'];
+let myMealRecipeArray = Object.values(myMealRecipe);
+let ingredientFinal = "";
+
+for (let i = 0 ; i < propertyCaption.length ; i++) { 
+  console.log(propertyCaption[i] + ": " + myMealRecipeArray[i]); 
```

Comment:
Object.values is not necessarily sorted. Therefore  it is possible that you get `[2, 'pizza', [..]]` instead. Which means you can possibly get `Meal name:  2`. You should be careful how you use it.